index.js

```diff
@@ -0,0 +1,69 @@
+// global variables
+let searchCityForm = document.getElementById('searchCityForm');
+let errorMessage = document.getElementById('errorMessage');
+let city = document.getElementById('city');
+let temp = document.getElementById('temp');
+let dataWrapper = document.getElementById('dataWrapper');
+let weatherImg = document.getElementById('weatherImg');
+let fah;
+
+
+let convertTemp = (kelvin) => {
+
+    // For celcius
+    if(fah) {
+        fah = false;
```

Comment:
Use a switch-case block instead of the `fah` variable for determining what the next temperature display should be